core/tests/protractor_desktop/extensions.js

```diff
@@ -200,9 +203,13 @@ describe('Interactions', function() {
         }
         await explorationEditorPage.navigateToMainTab();
         await explorationEditorMainTab.deleteInteraction();
+        isChangesPresent = true;
+      }
+      if (isChangesPresent) {
```

Comment:
It seems like we could simplify this line to:

```suggestion
      if (interaction.testSuite.length > 0) {
```

Reply:
DONE